scan_test.go

```diff
@@ -30,16 +30,20 @@ import (
 	"github.com/uber-go/dosa"
 )
 
-type AllTypes struct {
-	dosa.Entity `dosa:"primaryKey=BoolType"`
-	BoolType    bool
-	Int32Type   int32
-	Int64Type   int64
-	DoubleType  float64
-	StringType  string
-	BlobType    []byte
-	TimeType    time.Time
-	UUIDType    dosa.UUID
+type AllTypesScanTestEntity struct {
```

Comment:
It took me good couple of hours to debug an issue in the walker because initially I only added the new types in AllTypes structure but apparently there were two AllTypes structures and TestParser test in (finder_test.go) would fail at random iterations. It was very confusing and painful to debug. This particular definition in scan_test.go isn't used anywhere else but it needed to be exact same structure as AllTypes defined in entity_parser_test.go.